digit-health-services/health-sync-service/src/test/java/org/digit/health/sync/service/FileStoreServiceTest.java

```diff
@@ -0,0 +1,75 @@
+package org.digit.health.sync.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.digit.health.sync.repository.ServiceRequestRepository;
+import org.digit.health.sync.service.checksum.MD5Checksum;
+import org.egov.tracer.model.CustomException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.http.*;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.util.ReflectionTestUtils;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+import java.io.IOException;
+import java.util.Collections;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MockitoExtension.class)
+class FileStoreServiceTest {
+
+    @Mock
+    private RestTemplate restTemplate;
```

Comment:
Use ServiceRequestRepository instead of using restTemplate directly

Reply:
Done